pkg/setting/setting_smtp.go

```diff
@@ -27,6 +28,7 @@ func (cfg *Cfg) readSmtpSettings() {
 	cfg.Smtp.FromAddress = sec.Key("from_address").String()
 	cfg.Smtp.FromName = sec.Key("from_name").String()
 	cfg.Smtp.EhloIdentity = sec.Key("ehlo_identity").String()
+	cfg.Smtp.EhloIdentity = sec.Key("startTLS_policy").String()
```

Comment:
This needs to be added to `/conf/default.ini` and `conf/sample.ini` and `docs/source/installation/configuration.md`

Reply:
Thanks. 
Added